### PR TITLE
Adds documentation for virtual bridge error

### LINKED ISSUE #182: Add documentation for resolving virtual bridge error when running qemu
I was facing this issue when trying to run the commands in `https://unikraft.org/community/hackathons/sessions/baby-steps/#02-building-and-running-the-httpreply-application`.

`access denied by acl file
failed to launch bridge helper
qemu-system-x86_64: -netdev bridge,id=br0: Device 'bridge' could not be initialized`

### PR DESCRIPTION
**Pull request by levihackerman-102**
Fixes #182

### CODE REVIEW

File: content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md [hunk 1]
```diff
@@ -99,6 +99,13 @@ oOo oOO| | | | |   (| | | (_) |  _) :_
 Listening on port 8123...
 ```
 
+In this step you may face an error : 
+```
```
Comment (StefanJum) [inline content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md:103]:
```suggestion
In this step you may face the following error:

```text
```

File: content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md [hunk 2]
```diff
@@ -99,6 +99,13 @@ oOo oOO| | | | |   (| | | (_) |  _) :_
...
+access denied by acl file
+qemu-system-ppc64: -netdev bridge,br=virbr0,id=net0: bridge helper failed
+```
+For resolving this, check if `/etc/qemu/bridge.conf` exists and if it does add the line `allow virbr0` to it to allow your bridge to run.
```
Comment (StefanJum) [inline content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md:107]:
```suggestion

To fix this, check if the `/etc/qemu/bridge.conf` file exists and if it does add the line `allow virbr0` to it to allow your bridge to run.
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
Thank you for this.

Besides the comments, please [sign-off your commit](https://unikraft.org/docs/contributing/submitting-changes/#developers-certificate-of-origin) and change the message to something more explicit (something like: `hack/sessions: Document qemu bridge acces denied error`).

Also, should deleting the `allow all` line form the `/etc/qemu/bridge.conf` file reproduce this issue? I tried doing that and reloading the `qemu-kvm` service but the command still worked.

**Comment by levihackerman-102** [comment]
@StefanJum made the required changes

**Comment by levihackerman-102** [comment]
> Also, should deleting the `allow all` line form the `/etc/qemu/bridge.conf` file reproduce this issue? I tried doing that and reloading the `qemu-kvm` service but the command still worked.

Interesting, it does create an issue on my system and seems to be a common problem. Not sure why it doesn't reproduce on your system.

File: content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md [hunk 3]
```diff
@@ -99,6 +99,14 @@ oOo oOO| | | | |   (| | | (_) |  _) :_
 Listening on port 8123...
 ```
 
+In this step you may face the following error:
+```text
+access denied by acl file
+qemu-system-ppc64: -netdev bridge,br=virbr0,id=net0: bridge helper failed
+```
+
+To fix this, check if the `/etc/qemu/bridge.conf` file exists. If it doesn't, then create it first.and then add the line `allow virbr0` to it to allow your bridge to run.
```
Comment (razvand) [inline content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md:108]:
```suggestion
To fix this, check if the `/etc/qemu/bridge.conf` file exists.
If it doesn't, then create it first and then add the line `allow virbr0` to it to allow your bridge to run.
```

Comment (razvand) [reply content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md:108]: @levihackerman-102, if you add `allow all`, would that work?

**Review by razvand — CHANGES_REQUESTED** [review]
@StefanJum, @levihackerman-102, I added a comment. I got this error in the past, but it's only for certain systems / use cases. Not sure what's causing some systems to require it or some others, such as @StefanJum's system, do not cause the issue.

File: content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md (hunk 3 above)
Comment (levihackerman-102) [reply content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md:108]: Yes @razvand, it works.

File: content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md [hunk 4]
```diff
@@ -99,6 +99,15 @@ oOo oOO| | | | |   (| | | (_) |  _) :_
 Listening on port 8123...
 ```
 
+In this step you may face the following error:
```
Comment (StefanJum) [inline content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md:102]:
```suggestion
In this step you may face the following error:

```

**Review by StefanJum — CHANGES_REQUESTED** [review]
One small comment. Besides this, please also [squash your commits](https://www.git-tower.com/learn/git/faq/git-squash/).

**Comment by levihackerman-102** [comment]
@StefanJum made the changes

**Comment by StefanJum** [comment]
Looks good now @levihackerman-102, just modify the commit message to contain only the description of the changes and your sign-off, then we can merge this.

**Comment by levihackerman-102** [comment]
Ahh my bad, fixed it @StefanJum

**Review by StefanJum — APPROVED** [review]
All good now, thanks @levihackerman-102.

Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Comment by StefanJum** [comment]
No longer required after updates to the Unikraft website.

**Pull request closed without merging**